composer.json

```diff
@@ -13,20 +13,20 @@
 	],
 	"require": {
 		"php": ">=7.1",
-		"composer/installers": "^1"
+		"composer/installers": "^1.12.0"
 	},
 	"require-dev": {
-		"automattic/vipwpcs": "^2.2",
-		"dealerdirect/phpcodesniffer-composer-installer": "^0.4.1 || ^0.5 || ^0.6.2 || ^0.7",
-		"php-parallel-lint/php-parallel-lint": "^1.0",
-		"phpcompatibility/phpcompatibility-wp": "^2.1",
-		"squizlabs/php_codesniffer": "^3.5",
+		"automattic/vipwpcs": "^2.3.3",
+		"dealerdirect/phpcodesniffer-composer-installer": "^v0.7.2",
+		"php-parallel-lint/php-parallel-lint": "^1.3.1",
+		"phpcompatibility/phpcompatibility-wp": "^2.1.3",
+		"squizlabs/php_codesniffer": "^3.6.2",
 		"wp-coding-standards/wpcs": "^2.3.0",
 		"yoast/wp-test-utils": "^1",
-		"phpstan/phpstan": "^1.4",
+		"phpstan/phpstan": "^1.4.6",
 		"phpstan/extension-installer": "^1.1",
 		"phpstan/phpstan-strict-rules": "^1.1",
-		"szepeviktor/phpstan-wordpress": "^1.0"
+		"szepeviktor/phpstan-wordpress": "^1.0.2"
```

Comment:
Are there particular bugs that you're trying to avoid with these new minimums?

The default `composer install`  / `composer update` would already install the latest versions with the existing constraints anyway.

Reply:
I prefer having tighter requirements, so everyone that's using/developing the plugin has a similar environment. I would even pitch the idea of getting rid of the carets on version numbers, but that would require us to keep an active eye on updates.

Reply:
Personally I update my deps very often. If there are features we want in the newer versions, we should probably go for them IMO.

Reply:
Whether you have a tight constraint or not, using the latest versions requires `composer update` often - so there's not really any value in using tight constraints, when the current constraints give exactly the same (latest) versions anyway. 

The issue you're trying to solve for, is a people-problem, not a technology problem.

Reply:
Fair point